schema_cache.rst

```diff
@@ -0,0 +1,78 @@
+.. _schema_cache:
+
+Schema Cache
+============
+
+PostgREST uses the database schema cache to get information for several of its features:
+
+- For :ref:`resource_embedding`, it needs relationships' information, such as related tables, foreign keys, cardinalities, etc.
+- For :ref:`s_procs`, it needs access to their metadata, for example, parameters, return type or volatility. It also needs to verify if the function is overloaded.
+- To do an :ref:`upsert`, it looks for the primary keys.
+- To do an INSERT, it also looks for the primary keys in order to return the Location header.
+- For OPTION requests, it verifies the existence and the data modification capabilities of a table or view.
+- For :ref:`open-api`, it needs information about objects in the database (e.g., object description, columns and parameters).

```

Comment:
Ditto for "e.g.":

```suggestion
- For :ref:`open-api`, it looks for `comments <https://www.postgresql.org/docs/current/sql-comment.html>`_ on the database objects.
```

"Database objects" could be further clarified. I don't recall precisely which objects comments we look for.

Reply:
Looking at [the code](https://github.com/PostgREST/postgrest/blob/1b12b112a1e491f51e48e37d5d828e4a390de26b/src/PostgREST/App.hs#L462-L469), the comments are accessed directly from the database. Tested it and there's no need to reload the schema cache to get the comments, but columns, PK, FK, INSTEAD OF triggers, and procedures metadata do need it.